e2e_testing/torchscript/basic.py

```diff
@@ -1014,3 +1014,67 @@ def forward(self, a, b):
 @register_test_case(module_factory=lambda: ReturnTwoTensorF32I64())
 def ReturnTwoTensorF32I64_basic(module, tu: TestUtils):
     module.forward(tu.rand(2, 3), torch.randint(5, (2, 3)))
+
+# ==============================================================================
+
+class MaxPool2dWithIndicesModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.max_pool2d_with_indices(x,
+                                                      kernel_size=[6, 8],
+                                                      stride=[2, 2],
+                                                      padding=[3, 4],
+                                                      dilation=2)
+
+@register_test_case(module_factory=lambda: MaxPool2dWithIndicesModule())
+def MaxPool2dWithIndicesModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(2, 4, 6, 8) - 0.5)
```

Comment:
You can specify a range as argument flags in `tu.rand`:

https://github.com/llvm/torch-mlir/blob/30df2ec71b8f7c6f5e0bd5bb26a62e70c0520b77/python/torch_mlir_e2e_test/torchscript/framework.py#L151

Same comment for the other e2e tests